src/page-response-impact/ResponseImpactDashboard.tsx

```diff
@@ -39,6 +43,8 @@ import {
 const ResponseImpactDashboard: React.FC = () => {
   const { data: localeDataSource } = useLocaleDataState();
   const [scenarioState] = useScenario();
+  const { rtData } = useContext(FacilityContext);
+  const useRt = useFlag(["useRt"]);
```

Comment:
@macfarlandian @jessex I'm not sure whether we would want this feature flag on the impact page, but I started with it here. Would we want to remove this section of the impact page if they don't select the use Rt functionality?

Reply:
I could be wrong but I think that Excalibur is probably an Rt-dependent feature in general and that we assume Rt is enabled in order to use it. That feature flag was a temporary measure to avoid breaking all other functionality while it was in flight and should go away in the near future, so I'd opt in favor of just removing this flag from here, unless @macfarlandian disagrees.

Reply:
yes that's correct, it was just a precaution to keep master release-safe while work was in progress. not intended to be a long-term thing and should probably be ripped out now that it has been successfully released. i'll make an issue so we don't forget

Reply:
Okay cool, I've removed it from the response impact page. I'm still referencing it in the function `useFacilitiesRtData(facilities, true)` on line 88 which would just need to be updated once we remove the flag.